SPEC/PUBSUB.md

```diff
@@ -10,30 +10,29 @@
 
 > Subscribe to a pubsub topic.
 
-##### `ipfs.pubsub.subscribe(topic, handler, [options], [callback])`
+##### `ipfs.pubsub.subscribe(topic, handler, [options])`
 
 - `topic: String`
 - `handler: (msg) => {}` - Event handler which will be called with a message object everytime one is received. The `msg` has the format `{from: String, seqno: Buffer, data: Buffer, topicIDs: Array<String>}`.
 - `options: Object` - (Optional) Object containing the following properties:
   - `discover: Boolean` - (Default: `false`) Will use the DHT to find other peers.
```

Comment:
Please could you please add a comment that this option is currently unimplemented?

Reply:
If `discover` was the only option and it's not implemented yet, it means that currently there are no options available. Right?

Reply:
Correct, but please leave this here as we need to document `onError` asap, which you can pass to the HTTP client for dealing with pubsub errors and disconnections.

Any chance you could send a PR for it?